python/phonenumbers/phonenumberutil.py

```diff
@@ -1955,6 +1955,33 @@ def number_type(numobj):
     return _number_type_helper(national_number, metadata)
 
 
+def identify_line_type(numobj):
```

Comment:
Thanks for the suggesting &ndash; having a way to get `PhoneNumberType` as a string makes sense, but I think it would be better off as a separate class method.  

#223 adds some methods to various enum-type classes &ndash; would that cover what you're after (in a combination like: 
`PhoneNumberType.to_string(number_type(numobj))`) ?

Reply:
thnx for reply